forte/data/data_pack.py

```diff
@@ -639,6 +654,19 @@ def set_audio(
         ap.set_cache(audio)
         ap.sample_rate = sample_rate
 
+    def add_audio(self, audio):
```

Comment:
Let's also add a warning message in `set_audio` that encourages users to use `add_audio` instead. https://github.com/asyml/forte/pull/891#discussion_r942677425